src/styles/cardDetails.css

```diff
@@ -0,0 +1,41 @@
+ul {
+  list-style-type: none;
+  display: grid;
+  padding-left: 5px;
+  width: 100%;
+}
+
```

Comment:
- [ ] Kindly use `Lato` font-family and `Gill Sans` font-family according to the template design. 👍

Reply:
- [ ] Please address the previous request: make sure to use `Lato` font-family and `Gill Sans` font-family according to the template design. 💯 😄